test/goldens/end2end/http-0.13.0.json_report.md

```diff
@@ -1,4 +1,4 @@
-## 20/20 Follow Dart file conventions
+## 30/20 Follow Dart file conventions
```

Comment:
30/20 looks too good to be true. Should this not be 30/30? :)

Reply:
I'm not sure why `makeSection` has a fixed `maxPoints` parameter, shouldn't we rather just use the sum of subsection.maxPoints instead?

Reply:
I vaguely remember there being some reason for this - but now I cannot remember why...